app/templates/media/index.hbs

```diff
@@ -232,7 +265,28 @@
             update=(action (mut text))
             placeholder=(t "media.index.search")
           }}
-        </div>
+        </span>
+
+        {{! Sort }}
+        <span class="sort-container">
+          {{#bootstrap/bs-dropdown as |dropdown|}}
+            {{#dropdown.button type="button--dark-outline" class="button--small" disabled=text}}
```

Comment:
Can we change this to something like `disabled=(not (is-empty text))` so it's a boolean check